### DISCUSSION TITLE
Simple group table

### DISCUSSION THREAD

**Discussion by a-rr** [vuejs/petite-vue#156] [Q&A]
I am struggle to get a simple grouped table to work with petite-vue

I am looking for something like:

|  Price | Car |
| Tesla
|  M3 | 10K |
| M-neo | 100K |
| GMC
| Somecar | 8K |

My data structure is already nested:
[ owner:"Tesla, models:[{name:"M3", price:"10K"},{name:"M-neo", price:"100K"},
[ owner:"GMC, models:[{name:"Somecar price:"8K"}],

And I can render them in other formats (like divs), tables, trs and tds are a non go. It gives me a "Uncaught TypeError: Cannot read properties of null (reading 'insertBefore')"

What I am doing wrong (or how could I solve this problem)?

https://jsfiddle.net/mkfgqw24/16/

**Comment by sqllyw** [comment]
use div for the outer loop, and table for the inner loop should work:
https://jsfiddle.net/sjuvy381/1/
